node/core/pvf/worker/src/executor_intf.rs

```diff
@@ -212,47 +205,31 @@ impl Executor {
 	///
 	/// Failure to adhere to these requirements might lead to crashes and arbitrary code execution.
 	pub unsafe fn execute(
-		&self,
```

Comment:
Why? Semantically, the executor shouldn't be consumed on usage

Reply:
`create_runtime_from_artifact` takes an owned `Config`, so we would have to clone it. And we are already cloning `Executor` when passing it in to the thread so it's guaranteed to be owned anyway. But it does seem like the optimization is not worth the potential confusion.